integration-test/src/test/java/org/cloudfoundry/client/v3/SpacesTest.java

```diff
@@ -48,10 +50,13 @@
 import org.junit.Ignore;
 import org.junit.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 import reactor.test.StepVerifier;
+import sun.misc.IOUtils;
```

Comment:
Don't use the `sun.misc` package.

You could use `org.springframework.util.StreamUtils` and change line 279 to something like:
```
byte[] manifest = StreamUtils.copyToByteArray(new ClassPathResource("test-manifest.yml").getInputStream());
```

Or you could do something like:
```
 File file = new ClassPathResource("test-manifest.yml").getFile();
 byte[] manifest = Files.readAllBytes(file.toPath());
```